File: Java/1672_RichestcustomerWealth.java
// 1672 - Richest Customer Wealth

public int maximumWealth(int[][] accounts) {
    int max = 0, s = 0;
    for(int i = 0; i < accounts.length; i++) {
        s = 0;
        for(int j = 0; j < accounts[i].length; j++) {
            s += accounts[i][j];
        }
        max = Math.max(max, s);
    }
    return max;
}

File: CPP/1672_RichestCustomerWealth.cpp
// 1672 - Richest Customer Wealth

int maximumWealth(vector<vector<int>>& accounts) {
    int r = 0, s = 0;
    for(int i = 0; i < accounts.size(); i++) {
        s = 0;
        for(int j = 0; j < accounts[i].size(); j++) {
            s += accounts[i][j];
        }
        r = max(r, s);
    }
    return r;
}